Keda.Cosmosdb.Scaler/src/Services/CosmosDbTrigger.cs

```diff
@@ -0,0 +1,73 @@
+﻿using System;
+using System.Data.Common;
+using Microsoft.Azure.Documents.ChangeFeedProcessor;
+
+
+namespace Keda.Cosmosdb.Scaler.Services
+{
+    internal class CosmosDbTrigger
+    {
+        public string DocDBConnectionString { get; internal set; }
+        public string LeasesDocDBConnectionString { get; internal set; }
+        public string DatabaseName { get; internal set; }
+        public string CollectionName { get; internal set; }
+        public string LeaseDatabaseName { get; internal set; }
+        public string LeaseCollectionName { get; internal set; }
+        public string LeaseCollectionPrefix { get; internal set; }
+        public string AccountName { get; internal set; }
+    }
+
+    internal class CosmosDbTriggerMetadata

```

Comment:
Just out of curiosity, if all of this is hardcoded anyway, why do we need it? Is it because you'll do a follow-up PR for the configuration?

Reply:
I think you need to expose these through config  via Options - similar to https://github.com/Azure/azure-webjobs-sdk-extensions/blob/dev/src/WebJobs.Extensions.CosmosDB/Config/CosmosDBOptions.cs